.github/workflows/ci.yml

```diff
@@ -157,10 +157,8 @@ jobs:
       timeout-minutes: 2
       with:
         trap-caching: false
+        languages: javascript
     
-    - uses: github/codeql-action/autobuild@v2
```

Comment:
The `github/codeql-action/autobuild@v2` step was added in #5737, and was needed for the C# Now that we're back to just JavaScript, we don't need it.